app/components/sdg_management/subnavigation_component.rb

```diff
@@ -0,0 +1,26 @@
+class SDGManagement::SubnavigationComponent < ApplicationComponent
+  attr_reader :current
+
+  def initialize(current:)
+    @current = current
+  end
+
+  private
+
+    def sections
+      %i[goals targets]
+    end
+
+    def link_to_section(section)
+      link_to "SDG::#{section.to_s.classify}".constantize.model_name.human(count: 2).titleize,
+              path_for(section),
+              "aria-selected": section == current
```

Comment:
It looks like this attribute is causing invalid HTML:

`Element a is missing one or more of the following attributes: aria-checked, role.`

Reply:
Finally, `aria-selected` attribute was replaced with the HTML class `is-active` which provides the same styles, and resultant markup is also valid.